packages/runtime/test-runtime-utils/src/test/insecureUrlResolverTest.spec.ts

```diff
@@ -22,6 +22,14 @@ describe("Insecure Url Resolver Test", () => {
     beforeEach(() => {
         resolver = new InsecureUrlResolver(hostUrl, ordererUrl, storageUrl, tenantId, bearer);
         request = resolver.createCreateNewRequest(fileName);
+
+        // Mocking window since the resolver depends on window.location.host
+        if (typeof window === "undefined" && typeof global === "object") {
+            // eslint-disable-next-line @typescript-eslint/dot-notation
+            global["window"] = {
+                location: { host:"localhost" } as unknown as Location,
+            } as unknown as (Window & typeof globalThis);
+        }
     });
```

Comment:
Mocking 'window.location' now requires some additional casting.

Previously, was "mocked" in every test case (below).  Lifted to 'belowEach()'.